src/packages/core/components/tree/context-menu/tree-context-menu.service.ts

```diff
@@ -78,16 +78,25 @@ export class UmbTreeContextMenuServiceElement extends UmbLitElement {
 			}
 			#action-modal {
 				position: absolute;
-				left: 300px;
+				left: var(--umb-section-sidebar-layout-width);
 				height: 100%;
 				z-index: 1;
 				top: 0;
-				width: 300px;
+				width: var(--umb-section-sidebar-layout-width);
 				border: none;
 				border-left: 1px solid var(--uui-color-border);
 				border-right: 1px solid var(--uui-color-border);
 				background-color: var(--uui-color-surface);
 			}
+
+			#action-modal h3 {
```

Comment:
Looks like all styling is duplicated here as well as [src/packages/core/components/section/section-sidebar-context-menu/section-sidebar-context-menu.element.ts](src/packages/core/components/section/section-sidebar-context-menu/section-sidebar-context-menu.element.ts) for `#action-modal`. Can we do something about that?

Reply:
Yeah, noticed that at the time, I'll see what can be done to clean it up

Reply:
I cannot find the usage of these files anywhere, looks like legacy — so I have removed them.